sql/sql_db.cc

```diff
@@ -1463,82 +1443,6 @@ my_bool rm_dir_w_symlink(const char *org_path, my_bool send_error)
 }
 
 
-/*
-  Remove .frm archives from directory
-
-  SYNOPSIS
-    thd       thread handler
-    dirp      list of files in archive directory
-    db        data base name
-    org_path  path of archive directory
-
-  RETURN
-    > 0 number of removed files
-    -1  error
-
-  NOTE
-    A support of "arc" directories is obsolete, however this
-    function should exist to remove existent "arc" directories.
-*/
-long mysql_rm_arc_files(THD *thd, MY_DIR *dirp, const char *org_path)
```

Comment:
The code in here will need to be added in to `mysql_upgrade_db` to ensure arc directories are removed during upgrade

Reply:
Hi @robinnewhouse, thanks for the review. Yes, I am aware that we need to add code to `mysql_upgrade` to remove existing arc directories, I am a bit busy right now, will resume working on this in a week.